controllers/index.js

```diff
@@ -1,80 +0,0 @@
-'use strict';
-var randomstring = require("randomstring");
-var UrlData = require('../models/db');
-
-module.exports = {
-    index: {
-        get(req, res) {
-            res.render('index', {
-                title: "TinyUrl"
-            });
-        },
-
-        post(req, res, next) {
-            var regexp1 = /(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/;
-            var urlmatch1 = req.body.longurl.match(regexp1);
-            var regexp2 = /^(?:[a-z0-9A-Z]{0}|[A-Za-z0-9]{7})$/;
-            var urlmatch2 = req.body.shorturl.match(regexp2);
-            var custom = false;
-
-            if (req.body.shorturl !== "") {
-                custom = true;
-            }
-
-            if (urlmatch1 !== null && urlmatch2 !== null) {
-
-                var inpx = req.body.longurl,
-                    inpy = req.body.shorturl;
-
-                //  query to check if short url exits
-                var query = UrlData.findOne({ 'longurl': req.body.longurl },
-                    function(err, doc) {
-                        if (err) {
-                            res.send(err)
-                        }
-
-                        if (doc === null) {
-
-                            // create short links
-                            if (custom === true) {
-                                var surl = req.body.shorturl;
```

Comment:
don't declare variable inside if statements always declare above so people know whats it about.